src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -210,6 +210,18 @@
   <data name="SARIF1019_RuleIdMustBePresentAndConsistent" xml:space="preserve">
     <value>In every result, at least one of the properties result.ruleId and result.rule.id must be present. If both are present, they must be equal.</value>
   </data>
+  <data name="SARIF1022_PreferMessageIdInsteadOfMessageText" xml:space="preserve">
+    <value>Prefer message.id instead of message.text</value>
```

Comment:
>Prefer message.id instead of message.text [](start = 11, length = 41)

"In result messages, prefer the use of message.id and message.arguments to message.text and message.markdown. This allows you to improve the message text (for example, fixing a typo) by editing the rule metadata, without affecting the contents of the result object. This is important in applications such as automated bug filing, where you must compare results between one run and the next to ensure that you only file bugs for new results. In addition, using message.id and message.arguments can make the SARIF file smaller, and it enables the use of SARIF's localization feature, which allows the same results to be displayed in multiple languages."